### PR TITLE
Removal of extra span

### PR DESCRIPTION
**Pull request by remedcu**
### Issue Reference
This PR addresses the Issue : Removal of Extra span

Importance: Low

### CODE REVIEW

**Comment by realslimshanky** [comment]
@remedcu please provide more information regarding the issue. Like from where to remove the span you are talking about.

**Comment by remedcu** [comment]
I have already removed that. And you can check the same in "Files Changed".

File: templates/home.html [hunk 1]
```diff
@@ -31,7 +31,6 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
         <span class="text">
             DISTRICT NEEDS &amp; COLLECTION CENTERS<br></span>
```
Comment (vrthra) [inline templates/home.html]: Shouldn't you be removing this span rather than the next span?

Comment (vrthra) [reply templates/home.html]: (See the reply to this on the other thread by @remedcu )

**Pull request merged**